pyproject.toml

```diff
@@ -51,6 +51,7 @@ openpyxl = ">=3.0.10"
 tables = ">=3.7.0"
 lxml = ">=4.7.1,<4.9.0"
 pyreadstat = ">=1.1.9"
+types_SQLAlchemy = ">=1.4.50"
```

Comment:
I think this should go in the `tool.poetry.dependencies` section as it is needed for type checking

Reply:
Fixed.